### DISCUSSION TITLE
Wox Plugin: ChatGPT

### DISCUSSION THREAD

**Discussion by Mnikley** [Wox-launcher/Wox#3938] [Plugin Share]
I developed a ChatGPT Plugin for Wox and wonder if you want to include it in the repository:
https://github.com/Mnikley/Wox.Plugin.ChatGPT

It runs a flask server to process the streamed request and display the answer in a openai-like html page, an openai API key is required

**Comment by qianlifeng** [comment]
Great plugin! We are planing a preview feature in v2 which will support preview chatgpt streamed output realtime in your case. Now python plugin host and preview feature is not ready, when it's ready, I'm happy to let you know.
